packages/syft/src/syft/service/request/request.py

```diff
@@ -545,7 +544,7 @@ def _run(
                 obj = self.mutate(obj, value=self.previous_value)
                 self.linked_obj.update_with_context(context, obj)
 
-            return Ok(SyftSuccess(message=f"{type(self)} Success"))
+            return SyftSuccess(message=f"{type(self)} Success")
```

Comment:
@rasswanth-s this might break things, as for all other _run methods we're wrapping with Ok.

Reply:
Resolved by [dd43d68](https://github.com/OpenMined/PySyft/pull/7897/commits/dd43d68b9c30fa5cab8ced182881510481381ea6)